public_html/assets/css/responsive.css

```diff
@@ -352,11 +428,16 @@
         font-size: 30px;
     }
 
+    .cbailylaw__footer-menu ul li:not(:last-child) {
+        margin-right: 25px;
```

Comment:
Although this CSS property may also be causing the slight rightward shift of the menu